.travis/deploy_npm

```diff
@@ -36,22 +36,50 @@ fi
 exe node -v
 exe npm  -v
 
+echo ">>> Installing dependencies..."
 exe npm install
-exe npm install --no-save @qooxdoo/compiler
-
-exe npx qx --version
-exe npx qx pkg install -v 
-#compile everything
-exe npx qx compile --config-file=compileServer.json --target=build 
-#exe npx qx compile --config-file=compile.json       --target=build 
-#remove unnecessary stuff
-#exe find ./apps/transpiled -type f -name '*.map' -delete
-#exe find ./apps/transpiled -type f -name '*.js'  -delete
-#copy source
+
+echo ">>> Installing github.com/qooxdoo/qooxdoo-compiler"
+exe [ -d ./qooxdoo-compiler ] && exe rm -rf ./qooxdoo-compiler
+exe git clone https://github.com/qooxdoo/qooxdoo-compiler.git --depth=1 --single-branch ./qooxdoo-compiler
+exe cd ./qooxdoo-compiler
+exe npm install
+QX=`pwd`/qx
+exe cd ..
+exe $QX --version
+
+echo ">>> Compiling & deploying server library with the compiler master using current qx_server library from NPM..."
+exe $QX compile --config-file=compile-server.json --target=build
+exe mkdir -p ./lib
+exe rm -rf ./lib/*
+exe cp -R compiled/build/qx_server ./lib
```

Comment:
This appears not to be doing what the echo said it would do. This is using the compiled qx_server, not the current one in NPM...???

Reply:
The compiler has an NPM dependeny on `@qooxdoo/framework`, which is of course only the qx_server package (not the other framework classes). So the initial build of the framwork from master is done with qx_server from NPM. In the second step, this qx_server package is swapped out with the newly brewed one, in order to test if the compiler still works and to actually build and deploy the framework.

Reply:
As I said in the other comment, it's actually too late here since any change that will break the compiler is already in master at that point. We need to move this kind of test to the earlier PR testing workflow.

Reply:
i think it's still worth doing, because as it's cyclic dependencies, there could be an incompatibility that breaks during deployment - eg if compiler/PR and a framework/PR are going out together, both could pass tests, and then compiler/PR is merged and the framework/PR will no longer compile.  

By running the compiler tests as part of framework release, we know that we're not going to break the released compiler or the new framework.

Reply:
i'm just testing a version that does this now